.github/workflows/test-dev-stability.yml

```diff
@@ -52,4 +52,4 @@ jobs:
           composer-options: "--prefer-dist"
 
       - name: "Run PHPUnit"
-        run: "SYMFONY_DEPRECATIONS_HELPER=${{env.GITHUB_WORKSPACE}}/tests/baseline-ignore vendor/bin/phpunit"
```

Comment:
Purpose of this workflow is to alert us on upcoming deprecations by failing the build, instead of failing the main build and blocking our pipeline (often with deprecations that are not possible to fix by us).  Adding deprecations coming from doctrine/deprecations in this PR is fine, but changing the main workflow so that it fails the main build when there are deprecations will bring back the old problem. Rationale of all this is coming from https://github.com/orgs/doctrine/teams/doctrinecore/discussions/13?from_comment=5 People opening a PR shouldn't have their builds failed just because day before DBAL or some other package released a new version - that's completely unrelated to their changes they want to do in PR. Also, this cronjob is not limited to self and direct deprecations.

Reply:
makes sense :+1: Will adjust it

Reply:
Should this workflow then fail due to those indirect doctrine deprecations?